src/system_data.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python2
+
+import rospy
+import subprocess
+from std_msgs.msg import String
+
+def dataRead():
+    
+    ps_pub = rospy.Publisher('ps_stats', String, queue_size=20)
+    top_pub = rospy.Publisher('top_stats', String, queue_size=20)
+    tegra_pub = rospy.Publisher('tegra_stats', String, queue_size=20)
+    rospy.init_node('dataRead')
+
+    rate = rospy.Rate(10) #10 hz
+    while not rospy.is_shutdown():
+        
+
+        output = subprocess.check_output(['top','-bn1'])
+        top_pub.publish(output)
+
+        #need to run chmod +x so permission is not denied
+        subprocess.call(['chmod','+x','tegra.sh'])
```

Comment:
Need to get rid of this call to `chmod`, also the one below on line 27

Reply:
Why do we want to remove chmod? It was failing to run before I added those because it was giving me permission errors

Reply:
You only need to run `chmod` once to set the permissions, and then the presentations are changed forever, we don't have to do that every time we run the node